src/Mod/Import/App/ifc4.py

```diff
@@ -26588,25 +26588,25 @@ def fset( self, value ):
 class ifctasktimerecurring(ifctasktime):
 	'''Entity ifctasktimerecurring definition.
 
-	:param recurrance
-	:type recurrance:ifcrecurrencepattern
+	:param recurrence
```

Comment:
This highlights what I was trying to say in the chat - changes take time from the author, and they also take time from whoever reviews changes, and whoever reads those reviews, and whoever has to merge their changes with the same files...

So, we really are disussing a significant investment.